lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -544,17 +552,29 @@ def create_algorithm_sliders(self):
             "line_width",
             "Estimated spot width.",
             minimum=0.0,
-            maximum=15.0,
-            step_size=0.001,
+            maximum=15.0 * position_scale,
+            step_size=0.001 * position_scale,
             slider_type=ipywidgets.FloatSlider,
         )
         sigma_slider = self._add_slider(
             "Sigma",
             "sigma",
             "How much does the line fluctuate?",
-            minimum=1.0,
-            maximum=5.0,
-            step_size=0.001,
+            minimum=1.0 * position_scale,
+            maximum=5.0 * position_scale,
+            step_size=0.001 * position_scale,
             slider_type=ipywidgets.FloatSlider,
         )
-        return ipywidgets.VBox([thresh_slider, line_width_slider, window_slider, sigma_slider])
+        vel_calibration = position_scale / calibrated_kymo_channel.line_time_seconds
+        vel_slider = self._add_slider(
+            "Velocity",
+            "vel",
+            "How fast does the particle move?",
+            minimum=-5.0 * vel_calibration,
+            maximum=5.0 * vel_calibration,
+            step_size=0.001 * vel_calibration,
+            slider_type=ipywidgets.FloatSlider,
+        )
```

Comment:
is this a brand new widget component? if so maybe should be mentioned in the changelog?

Reply:
It is. It was found during WIT that for tubules this parameter is often quite critical. And you are right. I'll spin it into its own commit with a changelog entry.